packages/client/src/util/environment.ts

```diff
@@ -21,12 +22,9 @@ export function getEnvironment(): Environment {
       return {
         apiKey: process.env.XATA_API_KEY ?? getGlobalApiKey(),
         databaseURL: process.env.XATA_DATABASE_URL ?? getGlobalDatabaseURL(),
-        branch:
-          process.env.XATA_BRANCH ??
-          process.env.VERCEL_GIT_COMMIT_REF ??
-          process.env.CF_PAGES_BRANCH ??
-          process.env.BRANCH ??
-          getGlobalBranch(),
+        branch: process.env.XATA_BRANCH,
+        envBranch:
+          process.env.VERCEL_GIT_COMMIT_REF ?? process.env.CF_PAGES_BRANCH ?? process.env.BRANCH ?? getGlobalBranch(),
```

Comment:
`getGlobalBranch` returns global of `XATA_BRANCH`. I believe it should be on the other property not this one.